repose-aggregator/components/filters/scripting-filter/pom.xml

```diff
@@ -1,142 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have no idea. I didn't delete it, but it showed up when i rebased to the 8 branch.